curriculum/class-00/lab-a/starter-code/js/app.js

```diff
@@ -28,39 +28,42 @@ function makeRandom() {
   return Math.floor(Math.random() * names.length);
 }
 
-function displayPics(){
-  while(viewed.length < 6){
-    var rando = makeRandom();
-    while(!viewed.includes(rando)){
+function displayPics() {
+  while (viewed.length < 6) {
+    let rando = makeRandom();
+    while (!viewed.includes(rando)) {
       viewed.push(rando);
     }
+    newFunction(rando);
   }
-  console.log(rando);
-  // TODO: In a sentence or two, explain why the previous line of code threw an error when we changed the variable declaration from `var to `let`.
+  // TODO: In a sentence or two, explain why the previous line of code threw an error when we changed the letiable declaration from `let to `let`.
   // PUT YOUR RESPONSE IN THIS COMMENT
```

Comment:
dont forget to leave comments when asked for them, this is where we get to see if you understand why you did what you did